configs/nafnet/README.md

```diff
@@ -0,0 +1,90 @@
+# NAFNet (ECCV'2022)
+
+> [Simple Baselines for Image Restoration](https://arxiv.org/abs/2204.04676)
+
+> **Task**: Image Restoration
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Although there have been significant advances in the field of image restoration recently, the system complexity of the state-of-the-art (SOTA) methods is increasing as well, which may hinder the convenient analysis and comparison of methods. In this paper, we propose a simple baseline that exceeds the SOTA methods and is computationally efficient. To further simplify the baseline, we reveal that the nonlinear activation functions, e.g. Sigmoid, ReLU, GELU, Softmax, etc. are not necessary: they could be replaced by multiplication or removed. Thus, we derive a Nonlinear Activation Free Network, namely NAFNet, from the baseline. SOTA results are achieved on various challenging benchmarks, e.g. 33.69 dB PSNR on GoPro (for image deblurring), exceeding the previous SOTA 0.38 dB with only 8.4% of its computational costs; 40.30 dB PSNR on SIDD (for image denoising), exceeding the previous SOTA 0.28 dB with less than half of its computational costs.
+
+<!-- [IMAGE] -->
+
+<div align=center >
+ <img src="https://user-images.githubusercontent.com/43229734/199919292-81d307d9-144b-4d07-9f26-0c09d86e84a5.jpg" width="400"/>
+</div >
+
+## Results and models
+
+|                            Method                            | image size |    GoPro PSNR    |   GoPro SSIM   |    SIDD PSNR     |   SIDD SSIM    | GPU Info |                            Download                            |
+| :----------------------------------------------------------: | :--------: | :--------------: | :------------: | :--------------: | :------------: | :------: | :------------------------------------------------------------: |
+| [nafnet_c64eb2248mb12db2222_8xb8-lr1e-3-400k_sidd](/configs/nafnet/nafnet_c64eb2248mb12db2222_8xb8-lr1e-3-400k_sidd.py) |  256X256   |        -         |       -        | 40.3045(40.3045) | 0.9253(0.9614) | 1 (A100) | [model](https://download.openmmlab.com/mmediting/nafnet/NAFNet-SIDD-midc64.pth) \| log(coming soon) |
+| [nafnet_c64eb11128mb1db1111_8xb8-lr1e-3-400k_gopro](/configs/nafnet/nafnet_c64eb11128mb1db1111_8xb8-lr1e-3-400k_gopro.py) |  1280x720  | 33.7246(33.7103) | 0.9479(0.9668) |        -         |       -        | 1 (A100) | [model](https://download.openmmlab.com/mmediting/nafnet/NAFNet-GoPro-midc64.pth) \| log(coming soon) |
+
+Note:
+
+- PSNR is evaluated on RGB channels.
+- SSIM is evaluated by averaging SSIMs on RGB channels, however the original paper uses the 3D SSIM kernel.
```

Comment:
- a(b) where a denotes the value run by MMEditing, b denotes the value copied from the original paper.